### DISCUSSION TITLE
Choosing which upgrades to install.

### DISCUSSION THREAD

**Discussion by aoqia194** [ChrisTitusTech/winutil#496] [Ideas]
So I found this tool which is amazing. The only caveat I had was when using Upgrade Installs. I want to use it but I can't because there are about 3 applications I purposely need downgraded (and running Upgrade Installs will upgrade them regardless).

Maybe you could add a prompt (Y/N) for each update? (maybe a seperate upgrade installs button)

**Comment by DeveloperDurp** [comment]
Currently the only way to select which ones get updated is check the applications you wish to update and click "Start Install" this will force winget to install the latest version. 

Upgrade Installs will force an upgrade on all applications detected by winget (including ones installed outside of this tool)
